src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,39 @@
+@import "utils/variables";
+@import "utils/keyframes";
+@import "utils/mixins";
+@import "utils/extends";
+
+.watch {
+  @include watch-cize;
+  position: relative;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.